### PR TITLE
feat: Pbi 25 view current team profile backend

### PR DESCRIPTION
**Pull request by ferdifdi**
### Describe your changes
Implement view current team profile backend

### Backlog name
Pbi 25 view current team profile backend

### Acceptance criteria
- Please copy acceptance criteria from backlog info

### Example Request
```
curl --location --request GET 'localhost:8000/team-management/current/' \
--header 'Authorization: Token 3c8f298a25107c2d803d5f1f78d480425f2e4752'
```
```
{
    "id": 2,
    "name": "Ferdifadillah3",
    "logo": null,
    "description": "",
    "teammember": [
        2
    ]
}
```

### Checklist
- [X] I have performed a self-review of my code
- [X] Code successfully run on local
- [X] Feature / changes tested on local
- [X] No failing unit test
- [ ] Passed UAT Test
- [ ] Sonarqube tested 
- [ ] Required any changes to environment variable

### CODE REVIEW

File: team_management/serializers.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        user = TeamUserSerializers()
+        model = TeamMember
+        fields = [
+            'team', 
+            'user'
+        ]
+
+class TeamManagementSerializers(serializers.ModelSerializer):
+    class Meta:
+        teammember = TeamMemberSerializers(many=True)
```
Comment (andrew-susanto) [inline team_management/serializers.py:27]:
```suggestion
    teammember = TeamMemberSerializers(many=True)
    
        class Meta:
```

File: team_management/serializers.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        fields = [
+            'username', 
+            'email', 
+            'first_name', 
+            'last_name'
+        ]
+
+class TeamMemberSerializers(serializers.ModelSerializer):
+    class Meta:
+        user = TeamUserSerializers()
```
Comment (andrew-susanto) [inline team_management/serializers.py:18]:
```suggestion
    user = TeamUserSerializers()
    
        class Meta:
```

File: team_management/views.py [hunk 3]
```diff
@@ -31,3 +33,10 @@ def create(self, request, *args, **kwargs):
 			return Response(data=serialized_obj.data, status=status.HTTP_201_CREATED)
 			
 		return Response(data={"error": "Please make sure your [team name] is exist!"}, status=status.HTTP_400_BAD_REQUEST)
+
+	@action(detail=False,methods=["GET"])
```
Comment (luthfifahlevi) [inline team_management/views.py:37]: keknya harusnya detail=True,
trs tambahin self.object = self.get_objects()?
cc: @andrew-susanto wkwk

File: team_management/serializers.py (hunk 1 above)
Comment (ferdifdi) [reply team_management/serializers.py]: Done

File: team_management/serializers.py (hunk 2 above)
Comment (ferdifdi) [reply team_management/serializers.py]: Done

File: team_management/views.py (hunk 3 above)
Comment (andrew-susanto) [reply team_management/views.py:37]: owhh ini gua kepikirannya buat return current team aja sih fii

Comment (luthfifahlevi) [reply team_management/views.py:37]: oalah oke2 wkwk thank you.
Silakan merge @ferdifdi

Comment (ferdifdi) [reply team_management/views.py:37]: @andrew-susanto @luthfifahlevi bantu approve dulu wkwkwk

**Review by luthfifahlevi — APPROVED** [review]

**Review by andrew-susanto — APPROVED** [review]

**Pull request merged**